tests/config/test-environment.js

```
@@ -28,40 +28,30 @@ export const getDatabaseConfig = (testType) => {
-            console.log(`📁 Created data directory for integration tests: ${dataDir}`);
-          }
-        } catch (error) {
-          console.warn('⚠️ Could not create data directory:', error.message);
-        }
-        
-        return 'file:./data/test-integration.db';
+
+        // Default to in-memory for better isolation and performance
+        // File-based databases cause lock contention with parallel workers
+        return ':memory:';
       };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)